massgov/mayflower-patternlab
# Dp 8612,8730,8738 elected official updates

## Description

- Adds action finder variation to have a green title bar
- Adds contact us 'column' variant to have an alternative layout 
- Adds a contact list 'image' variant
- Refactors banner image for linked banner variation to avoid gaps between text area background and image as content length changes.


## Related Issue / Ticket

- [DP-8730 action finder color change](https://jira.state.ma.us/browse/DP-8730)
- [DP-8612 contact change on elected officials page](https://jira.state.ma.us/browse/DP-8612)
- [DP-8739 banner image alignment](https://jira.state.ma.us/browse/DP-8739)

## Steps to Test


1. View Elected officials page
2. Ensure action finder is green
3. Ensure contact row looks as expected across various widths (see https://jira.state.ma.us/browse/DP-8612)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
